prototyping/xarray/xdose.py

```diff
@@ -221,3 +222,102 @@ def _validate_coord_system(coord_system):
                 "\n".join(VALID_COORD_SYSTEMS)
             )
         )
+
+
+def plot_xdose_tcs_at_point(xdose_to_plot, point, coord_system="S"):
```

Comment:
To encourage the writing of tests during this experimentation stage did you want to be putting these modules within experimental?